fflonk_verifier/src/verifier.rs

```diff
@@ -1,32 +1,27 @@
 pub use crate::utils::{get_domain_size, get_omegas, get_proof, get_pubSignals, Omegas, Proof, ProofWithPubSignal};
 use ark_bn254::{
-    g1, g1::Parameters, Bn254, Fq, FqParameters, Fr, FrParameters, G1Projective, G2Projective,
+     g1::Parameters, Bn254, Fq, Fr, FrParameters, G1Projective,
 };
-use ark_bn254::{g2, Fq2, Fq2Parameters, G2Affine};
+use ark_bn254::{ Fq2, G2Affine};
 use ark_ec::short_weierstrass_jacobian::GroupAffine;
 use ark_ec::*;
 use ark_ff::{
-    field_new, Field, Fp256, Fp256Parameters, Fp2ParamsWrapper, One, PrimeField, QuadExtField,
-    UniformRand, Zero,
+    Field, Fp256, One, Zero,
 };
-use ark_poly::{domain, Polynomial};
-use core::num;
-use std::fmt::{format, Debug, DebugMap, Display};
-use std::hash::Hash;
-use std::marker::PhantomData;
-use std::ops::{Add, Mul, Neg, Sub};
+use std::ops::{Add, Mul, Sub};
 use std::str::FromStr;
 use std::vec;
-use crate::utils::get_proog_bigint;
+use crate::utils::{get_fr_from_bytes, get_proog_bigint, padd_bytes32};
 use num_bigint::*;
 
 use tiny_keccak::{Hasher, Keccak};
-use num_bigint::BigUint;
-
 
 pub type G1Point = <Bn254 as PairingEngine>::G1Affine;
 pub type G2Point = <Bn254 as PairingEngine>::G2Affine;
 
+
+const VALUES_TO_SKIP: usize = 8;
```

Comment:
Can we please add a comment on it's relevance? It is used in multiple places here and I'm missing a bit of context.

Reply:
done